.github/workflows/ci.yml

```diff
@@ -59,6 +29,3 @@ jobs:
 
       - name: Lint implementation
         run: flake8
```

Comment:
Do we even need to run flake8? The only python code left is the typing-summary script in scripts.

Reply:
On the other hand, linting can't hurt, and who knows whether we add more scripts later. So I retract my comment.